documentation/ImplementingASemanticCheck.md

```diff
@@ -216,12 +216,12 @@ find all of the `evaluate::ActualArgument` nodes.
 
 Note that the compiler has multiple types called `Expr`.  One is in the
 `parser` namespace.  `parser::Expr` is defined in the file
-`include/flang/parser/parse-tree.h`.  It represents a parsed expression that
+`include/flang/parser/Parse-tree.h`.  It represents a parsed expression that
```

Comment:
That looks wrong to me.

Reply:
The only time I did not use sed, I got it wrong. Thanks.

Reply:
Done